package/LICENSE

```diff
@@ -1,21 +0,0 @@
-    MIT License
```

Comment:
The LICENSE file now gets copied from the root of the repo so that there's not multiple copies: https://github.com/microsoft/react-native-winrt/blob/dunhor/autolinking/scripts/build-npm-package.cmd